prov/efa/src/dgram/efa_ep.c

```diff
@@ -439,37 +252,18 @@ static int efa_ep_enable(struct fid_ep *ep_fid)
 		attr_ex.recv_cq = ibv_cq_ex_to_cq(ep->scq->ibv_cq_ex);
 	}
 
-	attr_ex.cap.max_inline_data = ep->base_ep.domain->device->efa_attr.inline_buf_size;
+	attr_ex.cap.max_inline_data =
+		ep->base_ep.domain->device->efa_attr.inline_buf_size;
 
-	if (EFA_EP_TYPE_IS_RDM(ep->base_ep.domain->info)) {
-		attr_ex.qp_type = IBV_QPT_DRIVER;
-		attr_ex.comp_mask = IBV_QP_INIT_ATTR_PD | IBV_QP_INIT_ATTR_SEND_OPS_FLAGS;
-		attr_ex.send_ops_flags = IBV_QP_EX_WITH_SEND;
-		if (efa_domain_support_rdma_read(ep->base_ep.domain))
-			attr_ex.send_ops_flags |= IBV_QP_EX_WITH_RDMA_READ;
-		attr_ex.pd = ibv_pd;
-	} else {
-		assert(EFA_EP_TYPE_IS_DGRAM(ep->base_ep.domain->info));
-		attr_ex.qp_type = IBV_QPT_UD;
-		attr_ex.comp_mask = IBV_QP_INIT_ATTR_PD;
-		attr_ex.pd = ibv_pd;
-	}
+	assert(EFA_EP_TYPE_IS_DGRAM(ep->base_ep.domain->info));
```

Comment:
With this chang, I think the function should be renamed to `efa_dgram_ep_enable`

Reply:
Fixed